COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/CoordinateSystem.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class CoordinateSystem
+    {
+        public List<Point> listOfPoints = new List<Point>(); //List with all Points
+
+        public List<double> listOfDistsPoints = new List<double>(); //List with all Distances between the points
+
+        public void Add(Point point)
+        {
+            listOfPoints.Add(point);
+        }
+
+        public void DisplayAllPoints()
+        {
+            int count = 1;
+
+            foreach (Point i in listOfPoints)
+            {
+                Console.Write($"point {count++} : {i};\n");
+            }
+        }
+
+        public void FindDistance()
+        {
+            foreach (Point k in listOfPoints) // TODO потрібно створити якусь копію ліста, звідки видаляти член K після того, як він уже був прогнаний в циклі. Аби не дублювати ті самі відстані
+            {                                 //мб замість ліста тут краще використати черги/стаки???
+                foreach (Point l in listOfPoints)
+                {
+                    if (k == l)
+                    {
+                        Console.WriteLine("There were points with simillar coordinates or just one point");
+                    }
```

Comment:
It can only happen when you compare the point to itself, because Point is a class and here you compare reference which will be always different for two different points.

Reply:
whole logic was remaden so the comment is not actual....